File: c programming/practice set/pattern by loop/problem4.c
/*
1
23
456 for n=3
*/

#include <stdio.h>

int main()
{
    int n;
    int num = 1;
    if (n == 0)
    {
        printf("0");
    }
    else
    {

        printf("Enter the value:");
        scanf("%d", &n);
        for (int i = 1; i <= n; i++)
        {
            for (int j = 1; j <= i; j++)
            {
                printf("%d", num);
                num++;
            }
            printf("\n");
        }
        return 0;
    }
}

File: Python/chapter7/problem4.py
num = int(input("Enter the number: "))

if num <= 1:
    print(f"{num} is not a prime number!")
else:
    for i in range(2, num):
        if (num % i) == 0:
            print(f"{num} is not a prime number!")
            break
    else:
        # This else aligns with the for loop and only executes if no break occurs
        print(f"{num} is a prime number!")
